canonical-websites/www.ubuntu.com
# Change short-terms page to service terms

## Done

- Change short terms page to service terms page
- Add redirect for all short terms pages
- Replace hardcoded links to short-terms with service-terms

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/legal/service-terms](http://0.0.0.0:8001/legal/service-terms)
- Make sure page matches [copy doc](https://docs.google.com/document/d/1W0WJrXaIvC54MDPgS62xfgPhqJaSE7RcZ2mjIqz1Y2o/edit)
- Go to <http://0.0.0.0:8001/legal/short-terms> and make sure you are redirected to <http://0.0.0.0:8001/legal/service-terms>


## Issue / Card

Fixes #2861
